views/layouts/main.handlebars

```diff
@@ -1,55 +1,65 @@
-<!DOCTYPE html>
```

Comment:
@kentshaffer Do you want to review this one to make sure your toggle bar is still compatible?